grunttasks/generate-valid-styles.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var path = require('path');
+var stripSpecialChars = require('./lib/strip-special-chars');
+
+module.exports = function (grunt) {
+  'use strict';
+   grunt.registerMultiTask('generate-valid-styles', 'Generate the list of valid styles for translation', function () {
+    var validStylesRegex = /style=('|")[\s\S]*?('|")/ig;
+    var validStylesArray = [];
+
+    // iterate through .pot file and extract the styles
+    this.files[0].src.forEach(function (src) {
+      var contents = grunt.file.read(src);
+      if (validStylesRegex.test(contents)) {
+        validStylesArray = validStylesArray.concat(contents.match(validStylesRegex));
```

Comment:
@shane-tomlinson https://www.dropbox.com/s/7qgdk01hwnoifxh/Screenshot%202015-08-11%2015.50.03.png?dl=0
I am able to see that it finds all the styles in the .pot files. Or am i misunderstanding the problem?